tests/test_bot.py

```diff
@@ -2087,6 +2087,15 @@ async def make_assertion(url, request_data: RequestData, *args, **kwargs):
             api_kwargs={"chat_id": 2, "user_id": 32, "until_date": until_timestamp},
         )
 
+    async def test_business_connection_id_argument(self, bot, monkeypatch):
```

Comment:
can we check that for all methods that have that parameter with medium effort?

Reply:
our linting tools can check whether an argument is unused in a method, is that enough?

Reply:
mh, fair point. maybe a comment along the lines "test_official + linting covers most, so we only do a one-off test here"?